\documentclass[11pt]{article}
\usepackage[a4paper,margin=1in]{geometry}

\usepackage{fontawesome}
\usepackage{fontspec}

\setmainfont[UprightFont       = *Regular ,
             BoldFont          = *Bold ,
             ItalicFont        = *Italic ,
             BoldItalicFont    = *BoldItalic ,
             Ligatures=TeX]{Junicode Two Beta}


% Typesetting
\usepackage{microtype}

\usepackage{xcolor}

\usepackage{enumitem}



\renewcommand\labelitemi{\(\circ\)}
\renewcommand\labelitemii{\(\circ\)}


\begin{document}


\noindent\parbox{.3\textwidth}{\LARGE \textbf{\textls[35]{Richard Creswell}}}
\parbox{.3\textwidth}{~}
\parbox{.4\textwidth}{
\-\hspace{0cm}{\faEnvelopeO}~~richard.creswell@hertford.ox.ac.uk\\
\-\hspace{0cm}{\faWhatsapp}~~\texttt{\footnotesize +44 7491880443}~~or~~\texttt{\footnotesize +1 617-752-2508}\\
\-\hspace{0cm}{\faGithub}~~github.com/rccreswell\\
{ \faHome}~~Oxford, England
}

\vspace{0.6cm}

\noindent {\large \textbf{\textsc{\textls[50]{Research Interests}}}}\vspace{-2.75mm} \\
\rule{\textwidth}{0.4pt}
\vspace{-7mm}
\begin{itemize}[leftmargin=*]
\setlength{\itemsep}{2pt}
\setlength{\parskip}{0pt}
\setlength{\parsep}{0pt}
\item Statistical inference for challenging time series models.
\item Epidemiology and computational biology of infectious diseases.
\item Efficient inference for applied Bayesian nonparametrics.
\end{itemize}

 

\vspace{0.4cm}

\noindent {\large \textbf{\textsc{\textls[50]{Education}}}}\vspace{-2.75mm} \\
\rule{\textwidth}{0.4pt}
\vspace{0.1mm}
\noindent\parbox{.65\textwidth}{\raggedright \textit{{\fontspec[RawFeature=+swsh]{Junicode Two Beta}D}octor of Philosophy,} \textbf{Computer Science}}
\parbox{.35\textwidth}{\raggedleft (in progress; anticipated Q2, 2023)}
University of Oxford, Oxford, England

\vspace{0.4cm}

\noindent\parbox{.75\textwidth}{\raggedright \textit{Master of Science,} \textbf{Applied Mathematics}}
\parbox{.25\textwidth}{\raggedleft  ~}
Columbia University, New York, New York


\vspace{0.4cm}

\noindent\parbox{.75\textwidth}{\raggedright \emph{Bachelor of Science,} \textbf{Applied Physics \emph{summa cum laude}}}
\parbox{.25\textwidth}{\raggedleft  ~}
Columbia University, New York, New York



\vspace{0.6cm}

\noindent {\large \textbf{\textsc{\textls[50]{Research Positions}}}}\vspace{-2.75mm} \\
\rule{\textwidth}{0.4pt}
\vspace{0.1mm}
\noindent\parbox{.75\textwidth}{\raggedright \textbf{Doctoral Student}}
\parbox{.25\textwidth}{\raggedleft 2019 Oct.--present}
University of Oxford, Oxford, England
\vspace{-.25cm}
\begin{itemize}
\item Supervisor: Professor David Gavaghan.
\vspace{-.3cm}
\item Co-supervisors: Ben Lambert, Simon Tavener, Martin Robinson, Chon Lok Lei.
\vspace{-.3cm}
\item Statistical inference for time series models, particularly differential equation models arising in computational biology, and deterministic and stochastic models of the spread of infectious diseases.
\end{itemize}

\vspace{0.2cm}

\noindent\parbox{.75\textwidth}{\raggedright \textbf{Research Associate}}
\parbox{.25\textwidth}{\raggedleft 2017 July--2019 Sep.}
Massachusetts Host-Microbiome Center, Brigham \& Women's Hospital,\\ Harvard Medical School, Boston, Massachusetts
\vspace{-.25cm}
\begin{itemize}
\item Supervisor: Professor Georg Gerber.
\vspace{-.3cm}
\item Machine learning and Bayesian nonparametric models for time series of the gut microbiome.
\vspace{-.3cm}
\item Bioinformatic analysis of metagenomic data.
\end{itemize}

\vspace{0.2cm}


\noindent\parbox{.75\textwidth}{\raggedright \textbf{Undergraduate Research Assistant}}
\parbox{.25\textwidth}{\raggedleft 2014 May--2015 Jan.}
Columbia University, New York, New York
\vspace{-.25cm}
\begin{itemize}
\item Supervisor: Professor Irving Herman.
\vspace{-.3cm}
\item Time-dependent properties of luminescent nanoparticles passivated by graphene.
\end{itemize}

\vspace{0.6cm}


\noindent {\large \textbf{\textsc{\textls[50]{Teaching Experience}}}}\vspace{-2.75mm} \\
\rule{\textwidth}{0.4pt}
\vspace{0.1mm}
\noindent\parbox{.75\textwidth}{\raggedright \textbf{Teaching Demonstrator}}
\parbox{.25\textwidth}{\raggedleft 2020 Oct.--present}
University of Oxford, Oxford, England
\vspace{-.25cm}
\begin{itemize}
\setlength{\itemsep}{4pt}
\setlength{\parskip}{0pt}
\setlength{\parsep}{0pt}
\item I worked as a teaching assistant on the following modules:
\begin{itemize}
\setlength{\itemsep}{0pt}
\item SABS Software engineering (2020--2021, 2021--2022, 2022--2023).
\item SABS Mathematical modelling (Michaelmas 2020).
\item SABS Scientific computing (Hilary 2021).
\item SABS Simulated data and reproducible data analysis (Summer 2021).
\item UNIQ+ Machine Learning and Bayesian Inference training session (Summer 2021).
\end{itemize}
\item Two of the years that I worked on the software engineering module, I led the students in extending their open source software assignments into publishable research projects.
\end{itemize}



\vspace{0.2cm}

\noindent\parbox{.75\textwidth}{\raggedright \textbf{Mentorship}}
\parbox{.25\textwidth}{\raggedleft 2021 Apr.--present}
University of Oxford, Oxford, England
\vspace{-.15cm}
\begin{itemize}
\setlength{\itemsep}{4pt}
\setlength{\parskip}{0pt}
\setlength{\parsep}{0pt}
\item Co-supervisor for the following postgraduate students in Professor Gavaghan's research group:
\begin{itemize}
\setlength{\itemsep}{0pt}
\item Kit Gallagher (rotation, 2022).
\item Katherine Shepherd (rotation and PhD, 2022--).
\item Ioana Bouros (rotation and PhD, 2021--).
\end{itemize}
\end{itemize}


%\vspace{0.2cm}

%\noindent\parbox{.75\textwidth}{\raggedright \textbf{Script Writer}}
%\parbox{.25\textwidth}{\raggedleft 2013 Jan.--2014 Jan.}
%openlectures, New York, New York
%\vspace{-.25cm}
%\begin{itemize}
%\setlength{\itemsep}{4pt}
%\setlength{\parskip}{0pt}
%\setlength{\parsep}{0pt}
%\item This startup was producing concise, freely accessible online video lectures for high school students. I wrote scripts for various topics in the science and mathematics curriculum.
%\end{itemize}

\vspace{0.5cm}

\noindent {\large \textbf{\textsc{\textls[50]{Other Experience}}}}\vspace{-2.75mm} \\
\rule{\textwidth}{0.4pt}
\vspace{0.1mm}
\noindent\parbox{.75\textwidth}{\raggedright \textbf{Statistical Consultant}}
\parbox{.25\textwidth}{\raggedleft 2022 Aug.--}
Oxford University Innovation UKHSA COVID-19 Testing Evaluation
\vspace{-.25cm}
\begin{itemize}
\setlength{\itemsep}{4pt}
\setlength{\parskip}{0pt}
\setlength{\parsep}{0pt}
\item I worked as a statistical consultant for Oxford University Innovation (OUI) in a collaboration with Ernst \& Young (EY) to conduct an impartial retrospective analysis of the COVID-19 testing program in England, commissioned by the UK Health Security Agency (UKHSA).
\item Using techniques from causal inference and economic-epidemiological modelling, we evaluated the effects of testing activities on the transmission of COVID-19 from March 2020 to February 2022 and the cost-effectiveness of each aspect of the testing program.
\end{itemize}

\vspace{0.2cm}


\noindent\parbox{.75\textwidth}{\raggedright \textbf{Co-founder of Oxford Statistical Epidemiology Reading Group\\ \vspace{.1cm}}}
\parbox{.25\textwidth}{\raggedleft 2022 Oct.--}
\vspace{-.65cm}
\begin{itemize}
\setlength{\itemsep}{4pt}
\setlength{\parskip}{0pt}
\setlength{\parsep}{0pt}
\item I co-founded and organized the Oxford Statistical Epidemiology Reading Group, a biweekly journal club covering epidemiology, statistics, modelling, and related fields.
\end{itemize}


\vspace{0.2cm}


\noindent\parbox{.75\textwidth}{\raggedright \textbf{CoMo-DTC COVID-19 Collaboration Organizing Team}}
\parbox{.25\textwidth}{\raggedleft 2020 Oct.--2022 Mar.}
\vspace{-.65cm}
\begin{itemize}
\setlength{\itemsep}{4pt}
\setlength{\parskip}{0pt}
\setlength{\parsep}{0pt}
\item I joined the organizing team for the collaboration between Oxford's and Cornell's COVID-19 International Modelling Consortium (CoMo) and the Doctoral Training Centre (DTC) at Oxford.
\item Our work included investigating the development of high-quality software for CoMo's model of COVID-19 transmission, and implementing a hierarchy of compartmental transmission models for purposes of model comparison. 
\item A particular focus of our work was developing software, a web app, and pedagogical notebooks to help introduce newcomers to the field of epidemiological modelling.
\end{itemize}




\vspace{.2cm}


\noindent\parbox{.75\textwidth}{\raggedright \textbf{Bioinference Conference Organizing Committee\\ \vspace{.1cm}}}
\parbox{.25\textwidth}{\raggedleft 2021 Sep.--}
\vspace{-.65cm}
\begin{itemize}
\setlength{\itemsep}{4pt}
\setlength{\parskip}{0pt}
\setlength{\parsep}{0pt}
\item I worked on the organizing committees for the conferences ``Inference for Expensive Systems in Mathematical Biology'' held at Oxford on May 23--24, 2022, and its successor to be held at Oxford in Summer 2023.
\item To fund the conferences, the committee raised funds from from the London Mathematical Society, the Heilbronn Institute, and the Oxford Computer Science department.
\end{itemize}


\vspace{.2cm}

\clearpage


\noindent\parbox{.75\textwidth}{\raggedright \textbf{Co-founder of Shakespeare Appreciaton Society}}
\parbox{.25\textwidth}{\raggedleft 2022 Oct.--}
\vspace{-.65cm}
\begin{itemize}
\setlength{\itemsep}{4pt}
\setlength{\parskip}{0pt}
\setlength{\parsep}{0pt}
\item I assisted in the founding and operation of the Shakespeare Society at the University of Munich, which runs regular virtual screenings and discussions of Shakespeare's plays and other relevant literature.
\end{itemize}



\vspace{0.5cm}


\noindent {\large \textbf{\textsc{\textls[50]{Skills}}}}\vspace{-2.75mm} \\
\rule{\textwidth}{0.4pt}
\vspace{0.1mm}
\textbf{Programming:} Python, C, C++, R, MATLAB, Stan.\\
\textbf{Other computing:} MPI, Unix/Linux, SQL, Git, LSF, Slurm, AWS EC2, object-oriented programming,\\
\phantom{\textbf{Other computing: }}software testing, continuous integration. \\
\textbf{Design and web:} LaTeX, Blender, Illustrator, Inkscape, matplotlib, Plotly Dash, Flask.\\
\textbf{Other:} Office for National Statistics (ONS) Full accredited researcher.

\vspace{0.5cm}




\noindent {\large \textbf{\textsc{\textls[50]{Honors, Awards, and Funding}}}}\vspace{-2.75mm} \\
\rule{\textwidth}{0.4pt}
\vspace{-.65cm}
\begin{itemize}[leftmargin=*]
\setlength{\itemsep}{4pt}
\setlength{\parskip}{0pt}
\setlength{\parsep}{0pt}
\item EPSRC Doctoral Prize (2022)---\textbf{£27,221} funding to continue research at Oxford after finishing my DPhil.
\item Invited one-week research visit to Colorado State University, Fort Collins (2022).
\item Computer Science Scholarship (Oxford Department of Computer Science, 2019).
\item EPSRC Doctoral Training Partnership (2019).
\item Applied Physics Faculty Award (Columbia University, 2016).
\item C.\ Prescott Davis Scholar (Columbia University, 2016).
\end{itemize}

\vspace{0.5cm}



\noindent {\large \textbf{\textsc{\textls[50]{References}}}}\vspace{-2.75mm} \\
\rule{\textwidth}{0.4pt}
\vspace{-.65cm}
\begin{itemize}[leftmargin=*]
\setlength{\itemsep}{4pt}
\setlength{\parskip}{0pt}
\setlength{\parsep}{0pt}
\item David Gavaghan (Professor of Computational Biology, University of Oxford).\\ \hspace*{.5cm}david.gavaghan@dtc.ox.ac.uk

\item Ben Lambert (Senior Lecturer of Mathematics, University of Exeter).\\
\hspace*{.5cm}ben.c.lambert@gmail.com

\item Simon Tavener (Professor of Mathematics, Colorado State University).\\
\hspace*{.5cm}tavener@math.colostate.edu



\end{itemize}


\vspace{0.6cm}


%\clearpage


\noindent {\large \textbf{\textsc{\textls[50]{Publications and Presentations}}}}\vspace{-2.75mm} \\
\rule{\textwidth}{0.4pt}
\vspace*{.5cm}
\noindent\textbf{Journal papers}
\begin{itemize}[leftmargin=*]
\setlength{\itemsep}{4pt}
\setlength{\parskip}{0pt}
\setlength{\parsep}{0pt}
\vspace*{-.5cm}


\item {\color{gray}B.\ Lambert, C.\ L.\ Lei, M.\ Robinson, M.\ Clerx,} \textbf{R.\ Creswell,} {\color{gray} S.\ Ghosh, S.\ Tavener, and D.\ J.\ Gavaghan:} ``The impact of autocorrelated measurement processes on inference for ordinary differential equation models,'' \emph{Journal of the Royal Society Interface} (2023).


\item \textbf{R.\ Creswell,} {\color{gray} M.\ Robinson, D.\ Gavaghan, K.\ V.\ Parag, C.\ L.\ Lei, and B.\ Lambert:} ``A Bayesian nonparametric method for detecting rapid changes in disease transmission,'' \emph{Journal of Theoretical Biology}, vol.\ 558 (2023).

\item \textbf{R.\ Creswell,\textsuperscript{\dag}} {\color{gray}D.\ Augustin,\textsuperscript{\dag} I.\ Bouros,\textsuperscript{\dag} H.\ J.\ Farm,\textsuperscript{\dag} S.\ Miao,\textsuperscript{\dag} A.\ Ahern,\textsuperscript{\dag} M.\ Robinson, A.\ Lemenuel-Diot, D.\ Gavaghan, B.\ Lambert, and R.\ N.\ Thompson:} ``Heterogeneity in the onwards transmission risk between local and imported cases affects practical estimates of the time-dependent reproduction number,'' \emph{Philosophical Transactions of the Royal Society, A,} vol.\ 380 (2022).

\item {\color{gray} S.\ A.\ van der Vegt,\textsuperscript{\dag} L.\ Dai,\textsuperscript{\dag} I.\ Bouros,\textsuperscript{\dag} H.\ J.\ Farm,\textsuperscript{\dag} \textbf{\color{black}R.\ Creswell,\textsuperscript{\dag}} O.\ Dimdore-Miles,\textsuperscript{\dag} I.\ Cazimoglu, S.\ Bajaj, L.\ Hopkins, D.\ Seiferth, F.\ Cooper, C.\ L.\ Lei, D.\ Gavaghan, and B.\ Lambert:} ``Learning transmission dynamics modelling of COVID-19 using comomodels,'' \emph{Mathematical Biosciences,} vol.\ 349 (2022).

\item \textbf{R.\ Creswell,\textsuperscript{\dag}} {\color{gray}J.\ Tan,\textsuperscript{\dag} J.\ W.\ Leff, B.\ Brooks, M.\ A.\ Mahowald, R.\ Thieroff-Ekerdt, and G.\ K.\ Gerber:} ``High resolution temporal profiling of the human gut microbiome reveals consistent and cascading alterations in response to dietary glycans,'' \emph{Genome Medicine,} vol.\ 12 (2020).

\item {\color{gray} E.\ Bogart, \textbf{\color{black}R.\ Creswell,} and G.\ K.\ Gerber:} ``MITRE: inferring features from microbiota time-series data linked to host status,'' \emph{Genome Biology,} vol.\ 20 (2019).

\item {\color{gray} D.\ Zhang, D.\ Z.-R.\ Wang, \textbf{\color{black}R.\ Creswell,} C.\ Lu, J.\ Liou, and I.\ P.\ Herman:} ``Passivation of CdSe Quantum Dots by Graphene and MoS\textsubscript{2} Monolayer Encapsulation,'' \emph{Chemistry of Materials,} vol.\ 27, no.\ 14, pp.\ 5032--5039 (2015).
\end{itemize}
\hfill(\textsuperscript{\dag}$=$ joint first authorship.)

\vspace*{.4cm}
\noindent\textbf{Conference and workshop papers}
\vspace*{.5cm}
\begin{itemize}[leftmargin=*]
\setlength{\itemsep}{4pt}
\setlength{\parskip}{0pt}
\setlength{\parsep}{0pt}
\vspace{-.5cm}
\item \textbf{R.\ Creswell,} {\color{gray} M.\ K.\ Gibson, T.\ E.\ Gibson, J.\ W.\ Leff, and G.\ K.\ Gerber:} ``A multi-level Bayesian nonparametric model of longitudinal responses of the human microbiota to dietary interventions,'' \emph{ICML and IJCAI
Workshop on Computational Biology}, Stockholm, Sweden (2018).
\end{itemize}



%\vspace*{.4cm}
%\noindent\textbf{Letters}
%\vspace*{.5cm}
%\begin{itemize}[leftmargin=*]
%\setlength{\itemsep}{4pt}
%\setlength{\parskip}{0pt}
%\setlength{\parsep}{0pt}
%\vspace{-.5cm}
%\item {\color{gray}K.\ Gallagher,} \textbf{R.\ Creswell,} {\color{gray} and B.\ Lambert:} ``Identification and Attribution of Weekly Periodic Trends in Epidemiological Time Series Data,'' \emph{Journal} (2023).
%\end{itemize}


\vspace*{.4cm}
\noindent\textbf{Preprints}
\vspace*{.5cm}
\begin{itemize}[leftmargin=*]
\setlength{\itemsep}{4pt}
\setlength{\parskip}{0pt}
\setlength{\parsep}{0pt}
\vspace{-.5cm}


\item {\color{gray}R.\ Naidoo, B.\ Andersen-Waine, P.\ Dahal, S.\ Dickinson, B.\ Lambert, M.\ C.\ Mills,  C.\ Molyneux, E.\ Rowe, S.\ Pinto-Duschinsky, K.\ Stepniewska, R.\ Shretta, M.\ Voysey, M.\ Wanat, G.\ Yenidogan, L.\ J.\ White,} \textbf{and the EY-Oxford Health Analytics Consortium:} ``A multistage mixed-methods evaluation of the UKHSA testing response during the COVID-19 pandemic in England,'' medRxiv (2022).


\item {\color{gray}K.\ Gallagher,\textsuperscript{\dag} I.\ Bouros,\textsuperscript{\dag} N.\ Fan,\textsuperscript{\dag} E.\ Hayman,\textsuperscript{\dag} L.\ Heirene,\textsuperscript{\dag} P.\ Lamirande,\textsuperscript{\dag} A.\ Lemenuel-Diot, B.\ Lambert, D.\ J.\ Gavaghan,} \textbf{and R.\ Creswell:} ``Epidemiological Agent-Based Modelling Software (Epiabm),'' arXiv (2022).


\item \textbf{R.\ Creswell,} {\color{gray} B.\ Lambert, C.\ L.\ Lei, M.\ Robinson, and D.\ Gavaghan:} ``Using flexible noise models to avoid noise model misspecification in inference of differential equation time series models,'' arXiv:1410.5093 (2020).
\end{itemize}

\vspace*{.4cm}

\noindent\textbf{Talks}
\vspace*{.5cm}
\begin{itemize}[leftmargin=*]
\setlength{\itemsep}{4pt}
\setlength{\parskip}{0pt}
\setlength{\parsep}{0pt}
\vspace{-.5cm}
\item Inference for Expensive Systems in Mathematical Biology, Oxford, England (2022).
\item Microbiome Mini-Symposium (on the event of the visit of the Wageningen University Microbiology Laboratory to Harvard Medical School), Boston, Massachusetts (2019).
\item Forum for Advanced Biomedical Computation, Boston, Massachusetts (2018).
\item MIT-Harvard Microbiome Symposium, Cambridge, Massachusetts (2018).
\end{itemize}


\vspace*{.4cm}
\noindent\textbf{Poster presentations}
\vspace*{.5cm}
\begin{itemize}[leftmargin=*]
\setlength{\itemsep}{4pt}
\setlength{\parskip}{0pt}
\setlength{\parsep}{0pt}
\vspace{-.5cm}
\item The Royal Society, Modelling the Covid-19 Pandemic: Achievements and Lessons, London, England (2022).
\item Brigham \& Women's Hospital Pathology Research Celebration, Boston, USA (2019).
\item MIT-Harvard Microbiome Symposium, Cambridge, USA (2019).
\item ICML and IJCAI Workshop on Computational Biology, Stockholm, Sweden (2018).
\item Harvard Medical School Pathology Research Retreat, Boston, USA (2018).
\item MIT-Harvard Microbiome Symposium, Cambridge, USA (2018).
\item Computational Aspects of Biological Information, Microsoft Research New England, Cambridge, USA (2018).
\end{itemize}




\end{document}